types/node/http2.d.ts

```diff
@@ -558,13 +558,13 @@ declare module "http2" {
         prependOnceListener(event: "aborted", listener: (hadError: boolean, code: number) => void): this;
     }
 
-    export class Http2ServerResponse extends events.EventEmitter {
+    export class Http2ServerResponse extends stream.Stream {
         private constructor();
         addTrailers(trailers: OutgoingHttpHeaders): void;
         connection: net.Socket | tls.TLSSocket;
         end(callback?: () => void): void;
         end(data?: string | Buffer, callback?: () => void): void;
-        end(data?: string | Buffer, encoding?: string, callback?: () => void): void;
+        end(data?: string, encoding?: string, callback?: () => void): void;
```

Comment:
Not sure if we should keep this as it is allowed to pass `encoding` even for a `Buffer`. Users may have an API around this where they get `string | Buffer` and just forward which would be broken then.